core/frontend/src/test/Viewport.test.ts

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Can you name this file after what it specifically tests? Otherwise people will dump lots of other Viewport-related tests into it.

Reply:
I sort of thought I was "starting" a viewport/ScreenViewport test file, I'll go with "DeferredMarkerRemoval.test.ts" then